src/javascript/app_2/Stores/Modules/Trading/trade_store.js

```diff
@@ -396,6 +422,21 @@ export default class TradeStore extends BaseStore {
         }
     }
 
+    @action.bound
+    changeAllowEquals() {
+        const hasCallPutEqual = () => {
```

Comment:
it says `hasCallPutEqual` but it returns a object instead of true (if there is any contract with value equal to rise_fall_equal)
`.find` does not return a boolean